packages/shared-ux/chrome/navigation/src/ui/v2/components/navigation_group.tsx

```diff
@@ -80,10 +85,19 @@ function NavigationGroupComp(node: NodeProps) {
 
   return (
     <NavigationGroupContext.Provider value={contextValue}>
-      {/* Note: temporary UI. In future PR we'll have an EUI component here */}
-      <li style={{ paddingLeft: '20px', marginBottom: '15px' }}>{renderContent()}</li>
+      {renderContent()}
     </NavigationGroupContext.Provider>
   );
 }
 
-export const NavigationGroup = React.memo(NavigationGroupComp);
+function NavigationGroupComp(props: Props & NavigationBucketProps) {
+  if (props.preset) {
+    const { id, title, link, icon, children, ...rest } = props;
+    return <NavigationBucket {...rest} />;
```

Comment:
Looks like the `NavigationBucket` component is only used when we're using presets. Maybe this part of the conditional could also use `NavigationGroupInternalComp`, and we won't need the `NavigationBucket` component.

Reply:
It would add a lot of complexity to `NavigationGroupInternalComp`. `NavigationGroupInternalComp` and `NavigationGroupInternalComp` have different rendering logic and I prefer to keep those in different component for clarity.